src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionsController.kt

```diff
@@ -94,20 +97,40 @@ class IdentificationSessionsController(
             }
             .defaultIfEmpty(ResponseEntity.status(HttpStatus.NOT_FOUND).body(null))
             .doOnError { exception ->
-                log.error { "error occurred while getting the tc token for session with id $useIDSessionId; ${exception.message}" }
+                log.error { "error occurred while getting tc token for useIDSessionId $useIDSessionId;\n ${exception.message}" }
             }
             .onErrorReturn(
                 ResponseEntity.internalServerError().body(null)
             )
     }
 
     @GetMapping("/{eIDSessionId}")
-    fun getIdentity(@PathVariable eIDSessionId: UUID): Mono<IdentityAttributes> {
-        // Currently, mock identity
-        return Mono.just(IdentityAttributes("firstname", "lastname"))
-            .filter {
-                identificationSessionService.sessionExists(eIDSessionId)
+    fun getIdentity(@PathVariable eIDSessionId: UUID): Mono<ResponseEntity<GetResultResponseType>> {
+        return identificationSessionService.findByEIDSessionId(eIDSessionId)
+            .map {
+                eidService.getEidInformation(eIDSessionId.toString())
+            }
```

Comment:
since `getEidInformation` is (assuming) a blocking call, we could use `Mono.fromCallable` ?
https://betterprogramming.pub/how-to-avoid-blocking-in-reactive-java-757ec7024676

```kotlin
identificationSessionService.findByEIDSessionId(eIDSessionId)
.flatMap {
    Mono.fromCallable { eidService.getEidInformation(eIDSessionId.toString()) }
.subscribeOn(Schedulers.boundedElastic())
}
```